samples/HttpServer_WebSockets/app/application.cpp

```diff
@@ -10,6 +10,74 @@
 HttpServer server;
 int totalActiveSockets = 0;
 
+//Simplified container modelling a user session
+class CUserData
```

Comment:
Please put classes to extarnal files. It's important to keep samples as simple as possible.